source/_integrations/accuweather.markdown

```diff
@@ -19,6 +19,12 @@ ha_integration_type: service
 
 The AccuWeather integration uses the [AccuWeather](https://accuweather.com/) web service as a source for weather data for your location.
 
+<div class="note">
```

Comment:
I don't think this is fitting as a big note box in the introduction text of the integration. This could just as well be a normal paragraph added in the instructions below.

Please note, we have a snippet for the options flows you could use to help with navigating these paths for the reader.

../Frenck

Reply:
Agree - no doubt it was my initial frustration at the integration not doing the very thing I would have expected it to do. Thanks.

Reply:
Honestly I don't understand your frustration, at the end of setup, the integration displays this message

![obraz](https://github.com/home-assistant/home-assistant.io/assets/478555/cd5eb68b-1f06-4e87-bbeb-a1c272acd0b7)

But I agree that there should be information about this in the documentation as well.